src/host/README.md

```diff
@@ -0,0 +1,122 @@
+# src/host
+
+This directory implements the systems used by the `bin/host.js` command line tool.
+
+The `bin/host.js` tool has two commands. The main command operates a server and `bin/agent.js` instance to run test plans and the `plan` subcommand reads files from disk and packages them for the main command.
+
+A short invocation of the main command can be made with a list of files or file globs that make up a test plan. A test plan should contain tests for a single assistive technology to be tested. If for example all the reference files and test files are in two directories this command could be executed.
+
+```sh
+$ bin/host.js reference/** at/** >result.json
```

Comment:
I've been typing things like `./bin/host.js` all my life because I never knew that the presence of a forward slash influences how shells resolve the command. Thanks for the tip!